rhel/SPECS/grafana.spec

```diff
@@ -1,11 +1,5 @@
 %global debug_package   %{nil}
-%global provider        github
-%global provider_tld    com
-%global project         grafana
-%global repo            grafana
-# https://github.com/grafana/grafana
-%global import_path     %{provider}.%{provider_tld}/%{project}/%{repo}
-%global commit          v6.7.4
+%global commit          v7.1.0
```

Comment:
This should be 7.1.3. I refer to https://github.com/percona-platform/grafana/releases/tag/v7.1.3-pmm-2.9.2